defaults/manager_persistent_snapshots.yaml

```diff
@@ -3,78 +3,51 @@ aws:
-          scylla_product: "enterprise"
-          number_of_nodes: 5
-        'sm_20230702190638UTC':
-          keyspace_name: "5gb_sizetiered_5_2"
-          scylla_version: "5.2.3"
+          number_of_nodes: 3
+          # Recording cluster_id that is required for snapshots cleanup from the bucket in the future
+          cluster_id: "ba081f1b-1e46-4a3e-ad32-aaf393f0c758"
+        'sm_20240809154653UTC':
+          keyspace_name: "5gb_sizetiered_6_0"
+          scylla_version: "6.0.2"
           scylla_product: "oss"
-          number_of_nodes: 5
+          number_of_nodes: 3
+          cluster_id: "529700d6-ef2c-4420-8d3b-82f83d7ae0d7"
     10:
-      number_of_rows: 10485760
+      number_of_rows: 10737600
       expected_timeout: 3600  # 60 minutes
       snapshots:
-        'sm_20230223105105UTC':
-          keyspace_name: "10gb_sizetiered"
-          scylla_version: "5.1.6"
-          scylla_product: "oss"
-          number_of_nodes: 3
-        'sm_20230702173347UTC':
-          keyspace_name: "10gb_sizetiered_2022_2"
-          scylla_version: "2022.2.9"
-          scylla_product: "enterprise"
-          number_of_nodes: 4
-        'sm_20230702173940UTC':
-          keyspace_name: "10gb_sizetiered_2022_1"
-          scylla_version: "2022.1.7"
+        'sm_20240809143423UTC':
+          keyspace_name: "10gb_sizetiered_2024_2_0_rc1"
+          scylla_version: "2024.2.0~rc1"
           scylla_product: "enterprise"
-          number_of_nodes: 4
-        'sm_20230702173329UTC':
-          keyspace_name: "10gb_sizetiered_5_2"
-          scylla_version: "5.2.3"
+          number_of_nodes: 3
+          cluster_id: "b260dcbf-855a-4345-b5fc-09c1e6379a0c"
+        'sm_20240809154925UTC':
+          keyspace_name: "10gb_sizetiered_6_0"
+          scylla_version: "6.0.2"
           scylla_product: "oss"
-          number_of_nodes: 4
+          number_of_nodes: 3
+          cluster_id: "30944a61-a01d-4cee-b65f-cca11def7cf7"
     100:
-      number_of_rows: 104857600
+      number_of_rows: 107376000
       expected_timeout: 18000  # 300 minutes
       snapshots:
-        'sm_20230223130733UTC':
-          keyspace_name: "100gb_sizetiered"
-          scylla_version: "5.1.6"
-          scylla_product: "oss"
-          number_of_nodes: 3
-        'sm_20230702235739UTC':
-          keyspace_name: "100gb_sizetiered_2022_2"
-          scylla_version: "2022.2.9"
-          scylla_product: "enterprise"
-          number_of_nodes: 4
-        'sm_20230703000641UTC':
-          keyspace_name: "100gb_sizetiered_2022_1"
-          scylla_version: "2022.1.7"
+        'sm_20240812095953UTC':
+          keyspace_name: "100gb_sizetiered_2024_2_0_rc1"
+          scylla_version: "2024.2.0~rc1"
           scylla_product: "enterprise"
-          number_of_nodes: 4
-        'sm_20230703000157UTC':
-          keyspace_name: "100gb_sizetiered_5_2"
-          scylla_version: "5.2.3"
-          scylla_product: "oss"
-          number_of_nodes: 4
-    2048:
```

Comment:
is the 2tb isn't used ?

Reply:
That's actually a good question whether we need it since in the 3 hours [job](https://argus.scylladb.com/test/379b8fb7-0e42-4453-bc77-b857a8758986/runs?additionalRuns%5B%5D=c2048c63-a1d8-44a2-b0d9-8874c6d9b145) I was supposed to fix, 10GB backup is used.

24 hours [test](https://jenkins.scylladb.com/job/enterprise-2024.2/job/longevity/job/longevity-lwt-24h-test/) is disabled in CI.

Reply:
We have tests with  few days

Reply:
okay, I'll prepare such snapshot as well